<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Mission_Dialog</class>
 <widget class="QDialog" name="Mission_Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>272</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QPushButton" name="ReturnButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>58</y>
     <width>105</width>
     <height>48</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
   <property name="icon">
    <iconset resource="picture.qrc">
     <normaloff>:/new/picture/setting_interface/return.png</normaloff>:/new/picture/setting_interface/return.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>21</width>
     <height>14</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="StartButton">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>114</y>
     <width>448</width>
     <height>48</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/new/picture/mission_interface/Rectangle 250.png);
border-color: rgb(0, 150, 152);</string>
   </property>
   <property name="text">
    <string>启动任务</string>
   </property>
  </widget>
  <widget class="QPushButton" name="HistoryButton">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>170</y>
     <width>448</width>
     <height>48</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(0, 0, 0, 0);
border-color: rgb(0, 150, 152);
</string>
   </property>
   <property name="text">
    <string>查看任务</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>50</y>
     <width>480</width>
     <height>220</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/new/picture/mission_interface/mission_bakg.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>ReturnButton</zorder>
  <zorder>StartButton</zorder>
  <zorder>HistoryButton</zorder>
 </widget>
 <resources>
  <include location="picture.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>ReturnButton</sender>
   <signal>clicked()</signal>
   <receiver>Mission_Dialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>72</x>
     <y>81</y>
    </hint>
    <hint type="destinationlabel">
     <x>239</x>
     <y>135</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
